### DISCUSSION TITLE
There is no text-start neither text-end classes

### DISCUSSION THREAD

**Discussion by saifobeidat** [tailwindlabs/tailwindcss#7738] [Help] [answered]
I didn't find `text-start` neither `text-end` which will be converted to   `text-align: start; text-align: end;` and there's nothing in the docs about how to extend them
https://tailwindcss.com/docs/text-align

any idea?

**Comment by gabrielesilinic** [comment]
`text-left` `text-right` and `text-center` will do the job

**Reply by saifobeidat** [reply]
@gabrielesilinic in my case (multilingual website), I had to do this:

`ltr:text-left rtl:text-right` where as this can be done with only `text-start` if it existed

**Comment by adamwathan** [comment, marked as answer]
We’ve added these classes very recently and will be available in v3.1. Can install the `insiders` tag if you want to use now 👍🏻
